docs/source/settingup.rst

```diff
@@ -1,29 +1,90 @@
+
+Installation
 ------------
 
-- DataLad
+``datalad-osf`` is a Python package available on `pypi <https://pypi.org/project/datalad-osf/>`_ and installable via pip_.
 
-Before being able to use the extension, you need to have DataLad installed, which relies on `git-annex`_, `git`_ and `Python`_.
-If you don't have DataLad installed yet, please follow the instructions from `the datalad handbook <http://handbook.datalad.org/en/latest/intro/installation.html>`_.
+.. code-block:: bash
 
-- An account on the OSF
+   # create and enter a new virtual environment (optional)
+   $ virtualenv --python=python3 ~/env/dl-osf
+   $ . ~/env/dl-osf/bin/activate
+   # install from PyPi
+   $ pip install datalad-osf.
 
-You need an OSF account to be able to interact with it. If you don't have an account yet, `register here <https://osf.io/register>`_.
+If you want to use the most recent development version, clone its `GitHub repository <https://github.com/datalad/datalad-osf>`_ to your local computer and install the package in editable mode:
 
-- An account on a git repository hosting site
+.. code-block:: bash
 
-You should consider having an account on one or more repository hosting sites such as `GitHub <https://github.com/join>`__ , `GitLab <https://gitlab.com/users/sign_up>`_, `Bitbucket <https://bitbucket.org/account/signup/>`_ or similar"
+   # create and enter a new virtual environment (optional)
+   $ virtualenv --python=python3 ~/env/dl-osf
+   $ . ~/env/dl-osf/bin/activate
+   # install from local repository clone
+   $ pip install -r requirements-devel.txt
+   $ pip install -e .
 
-Installation
-------------
+Getting started
+^^^^^^^^^^^^^^^
+
+Here's the gists of some of this extension's functionality.
+Checkout the :ref:`Tutorial` for more detailed demonstrations.
+
+First, :ref:`provide your credentials <authenticate>`:
+
+.. code-block:: bash
+
+   # provide your OSF credentials, ideally as a token:
+   $ datalad osf-credentials
+   You need to authenticate with 'https://osf.io' credentials. https://osf.io/settings/tokens provides information on how to gain access
+   token: <your token here>
+   You need to authenticate with 'https://osf.io' credentials. https://osf.io/settings/tokens provides information on how to gain access
+   token (repeat): <your token here>
+   osf_credentials(ok): [authenticated as <user> <e-mail>]
+
+Next, create a sibling on the OSF for a DataLad dataset of your choice.
+Chose between different sibling modes to adjust how much of your dataset can be published and how it will be displayed, adjust whether your project should be private or public, attach additional meta data, or configure local sibling properties.
+The minimal example below will create a new (private) project with minimal metadata on the OSF and apply the necessary configurations to publish your complete dataset to it.
+
+.. code-block:: bash
+
+   # inside of a DataLad dataset
+   $ datalad create-sibling-osf --title best-study-ever -s osf
```

Comment:
`osf` is the default name, so it could be stripped

Reply:
thx, but I feel as if it would be helpful to be verbose here. I fear that else the ``--to osf`` is not as easily identified as a configurable, arbitrary name but as a generic way to refer to "the OSF"

Reply:
fine with me

Reply:
> thx, but I feel as if it would be helpful to be verbose here. I fear that else the --to osf is not as easily identified as a configurable, arbitrary name but as a generic way to refer to "the OSF"

I agree with this :+1: 

is `-s` the shortform? or is there a `--long_form_of_s`? If there is a long form, I think it'd be preferable to use here, because short forms are often abbreviations that can interrupt the reading flow.

Reply:
Its ``-s/--name``. I won't defend this decision at all costs, but the cheatsheet also mentions (only) the short form.
![image](https://user-images.githubusercontent.com/29738718/87638681-7dc53080-c744-11ea-83b7-6ee0532e789e.png)
And in this particular command, its also not easy to relate ``--title`` and ``--name`` to project versus sibling name. ``-s`` is more likely the sibling wherease ``--name`` is more ambiguous.

Reply:
true, fair points :+1: